content/integrations/pagerduty.md

```diff
@@ -18,7 +18,24 @@ You can also check out [this documentation][1] from Pagerduty.
 Once you have Pagerduty integrated, you can check out our custom [Pagerduty Incident Trends][2].
 {{< img src="integrations/pagerduty/pagerduty_incident_trends.png" >}}
 
+## FAQ
+### How do I automatically resolve a PagerDuty service when a monitor recovers?
+
+To automatically resolve a PagerDuty service when a monitor recovers
```

Comment:
This line just repeats the question. Let's remove it and begin with "You must include the PagerDuty notification..."